frontend/packages/kubevirt-plugin/src/components/modals/hardware-devices/HardwareDevicesModal.tsx

```diff
@@ -163,6 +163,7 @@ export const HardwareDevicesModal = withHandlePromise(
                     : t('kubevirt-plugin~No Host devices found')
                 }
                 isUserForbidden={isUserForbidden}
+                className="pf-c-empty-state"
```

Comment:
why ?

Reply:
we want the empty state will have a centralized text only when it's a modal, so this is a prop that should be inherited from the modal component IMO